### PR TITLE
Menu rework

### PR DESCRIPTION
**Pull request by juliensnz**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

This PR rework the index.html.twig base page and remove unused features. It also pefize the header and the base page. The first and the last commit are the most relevant.

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | NA
| Added Behats                      | NA
| Added integration tests           | NA
| Changelog updated                 | N
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require({
```
Comment (tamarasaurus) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/index.js]: Why two requires in the same file? Can this config be separated?

File: src/Oro/Bundle/RequireJSBundle/Resources/views/scripts.html.twig [hunk 2]
```diff
@@ -1,25 +0,0 @@
...
-    <script type="text/javascript" src="{{ asset(get_requirejs_build_path()) }}"></script>
-{% else %}
-    <script type="text/javascript" src="{{ asset('bundles/ororequirejs/lib/require.js') }}"></script>
-    <script type="text/javascript">
-        {{ get_requirejs_config() }}
-    </script>
-    <script type="text/javascript">
-        {% include 'PimUIBundle::requirejs.config.js.twig' %}
-        {% include 'PimNavigationBundle:Js:requirejs.config.js.twig' %}
-        {% include 'OroTranslationBundle::requirejs.config.js.twig' %}
```
Comment (tamarasaurus) [inline src/Oro/Bundle/RequireJSBundle/Resources/views/scripts.html.twig]: nice! 🔥 🔥 🔥

File: src/Pim/Bundle/EnrichBundle/Resources/views/index.html.twig [hunk 3]
```diff
@@ -1,96 +1,37 @@
...
+        <script type="text/javascript" src="bundles/ororequirejs/lib/require.js"></script>
+        <script type="text/javascript" src="js/require-config.js"></script>
+        <script type="text/javascript" src="bundles/pimenrich/js/index.js"></script>
+    </head>
+    <body>
+        <div id="page" class="AknDefault-page">
+            <div id="top-page" class="AknDefault-topPage">
+                <script>
+                    require(
+                        ['jquery', 'pim/form-builder'],
```
Comment (tamarasaurus) [inline src/Pim/Bundle/EnrichBundle/Resources/views/index.html.twig]: 😢 possible to put this js in a module instead?

File: src/Pim/Bundle/UserBundle/Resources/public/js/init-user.js [hunk 4]
```diff
@@ -1,40 +0,0 @@
-/* global require */
```
Comment (tamarasaurus) [inline src/Pim/Bundle/UserBundle/Resources/public/js/init-user.js]: is this not used anymore?

Comment (juliensnz) [inline src/Pim/Bundle/UserBundle/Resources/public/js/init-user.js]: removed :)

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/index.js (hunk 1 above)
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/index.js]: Yeah, we need it: first is config, second is requirements

**Comment by juliensnz** [comment]
@skeleton It's planned I will work on that right away :)

**Pull request merged**